### PR TITLE
feat(community-ui): start redesign community section interface

### PR DESCRIPTION
**Pull request by roger-rangel**

### CODE REVIEW

File: src/app/dashboard/mycommunity/page.tsx [hunk 1]
```diff
@@ -1,50 +1,66 @@
...
-import Link from 'next/link';
-import { Roboto } from 'next/font/google';
-import { Mixpanel } from '@/components/Mixpanel';
+import { useState, useEffect } from 'react';
 
-const roboto = Roboto({
-  weight: '300',
-  subsets: ['latin'],
-});
+import IntroCommunity from '../../../components/community/IntroCommunity';
```
Comment (neddy34) [inline src/app/dashboard/mycommunity/page.tsx:5]: Please create a `index.tsx` in `src/components/community` directory and export all components in there.
And import all components like

```ts
import { IntroCommunity, ...} from '@/components/community';
...
```

File: src/app/dashboard/mycommunity/page.tsx [hunk 2]
```diff
@@ -1,50 +1,66 @@
...
+import CreateCommunity from '../../../components/community/CreateCommunity';
+import JoinCommunity from '../../../components/community/JoinCommunity';
+import CommunitySection from '../../../components/community/CommunitySection';
 
 export default function MyCommunity() {
+  const [introModal, showIntroModal] = useState(true);
```
Comment (neddy34) [inline src/app/dashboard/mycommunity/page.tsx:11]: Please check how the other modal works like Create Collection, Mint NFT, Profile

File: src/components/Nft/TransferModal.tsx [hunk 3]
```diff
@@ -47,7 +47,7 @@ export function TransferModal({
           {nft.name}
         </h3>
         <div
-          // @ts-ignore
+          // @ts-ignored
```
Comment (neddy34) [inline src/components/Nft/TransferModal.tsx:50]: Please revert

File: src/components/community/CreateCommunity.jsx [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+import { useConnect } from '@connect2ic/react';
+import { Actor } from '@dfinity/agent';
+import { AssetManager } from '@dfinity/assets';
+import { Principal } from '@dfinity/principal';
+
+import { DFX_NETWORK } from '@/config';
+import { useActor } from '@/providers/ActorProvider';
+import { canisterId } from '@/declarations/vibeverse_assets';
+
+import ConnectSocials from '../../components/community/ConnectSocials';
```
Comment (neddy34) [inline src/components/community/CreateCommunity.jsx:13]: Import directly in the current folder

File: src/components/community/IntroCommunity.jsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styles from './IntroCommunity.module.scss';
```
Comment (neddy34) [inline src/components/community/IntroCommunity.jsx:2]: Well I agree with using css modules, but I prefer using tailwindcss.

File: src/components/community/JoinCommunity.jsx
Comment (neddy34) [inline src/components/community/JoinCommunity.jsx:1]: Please change modal behavior

**Review by neddy34 — CHANGES_REQUESTED** [review]
Overall looks good, added some changes regarding coding styles.

File: src/components/community/JoinCommunity.jsx
Comment (roger-rangel) [reply src/components/community/JoinCommunity.jsx:1]: What do you mean with "modal behaviour"??

File: src/components/community/IntroCommunity.jsx (hunk 5 above)
Comment (neddy34) [reply src/components/community/IntroCommunity.jsx:2]: Yes, we can use css modules, and what I mean was if tailwind is possible please use that. :)

**Review by neddy34 — APPROVED** [review]

**Pull request merged**